.changeset/selfish-trains-attack.md

```diff
@@ -0,0 +1,6 @@
+---
+'@twilio-paste/list': patch
+'@twilio-paste/core': patch
+---
+
+[List] add list style type to lists
```

Comment:
nitpick: could we say "add list-style-type **prop** to lists" here to be more explicit?